src/video/winrt/SDL_winrtvideo.cpp

```diff
@@ -256,7 +256,8 @@ static void WINRT_DXGIModeToSDLDisplayMode(const DXGI_MODE_DESC *dxgiMode, SDL_D
     SDL_zerop(sdlMode);
     sdlMode->w = dxgiMode->Width;
     sdlMode->h = dxgiMode->Height;
-    sdlMode->refresh_rate = dxgiMode->RefreshRate.Numerator / dxgiMode->RefreshRate.Denominator;
+    sdlMode->refresh_rate_numerator = dxgiMode->RefreshRate.Numerator / dxgiMode->RefreshRate.Denominator;
+    sdlMode->refresh_rate_denominator = 1;
```

Comment:
No, this needs to be:
```c
    sdlMode->refresh_rate_numerator = dxgiMode->RefreshRate.Numerator;
    sdlMode->refresh_rate_denominator = dxgiMode->RefreshRate.Denominator;
```

Reply:
it was already done in a previous commit ..